subworkflows/local/germline_variant_calling.nf

```diff
@@ -47,9 +49,10 @@ workflow GERMLINE_VARIANT_CALLING {
         .map{ meta, cram, crai, intervals, num_intervals ->
 
             //If no interval file provided (0) then add empty list
-            intervals_new = num_intervals == 0 ? [] : intervals
+            intervals_new  = num_intervals == 0 ? []            : intervals
+            intervals_name = num_intervals == 0 ? "no_interval" : intervals.simpleName
```

Comment:
we have had some issues with this in the past, where the intervals file name was not correctly resolved in some instances.

Reply:
it is not about the meta array, but that intervals.simpleName seems to be resolved at a different time or so. i don't know if it is a nextflow bug or intended behavior so far, but it has caused us a lot of headaches. Basically what has happened is this:
```
Just looking at the meta maps here I get once:
[patient:test2, sample:test5, gender:XX, status:0, id:test5_chr21_2-23354000]
[[patient:test2, sample:test5, gender:XX, status:0, id:test5_chr21_2-23354000]
and once
[patient:test2, sample:test5, gender:XX, status:0, id:test5_chr21_23355001-46709983]
[patient:test2, sample:test5, gender:XX, status:0, id:test5_chr21_2-23354000]
```

as you see sometimes the interval name is not properly resolved (first post, intended behavior is the second one, here we wanted to attach the intervals.baseName to the meta.id). however the files are always there correctly.

https://nfcore.slack.com/archives/C027CM7P08M/p1644242481203209?thread_ts=1644241819.942339&cid=C027CM7P08M

This also occured with a deep clone of the meta file: https://nfcore.slack.com/archives/C027CM7P08M/p1644248848566959?thread_ts=1644241819.942339&cid=C027CM7P08M

Once all your code is pushed i will pull it and run it with the test data I used there to see if anything suspicious happens. However, i am bit worried, that it could happen that accidentally the wrong intervals are combined possibly causing a silent error.